CMakeLists.txt

```diff
@@ -62,10 +67,31 @@ else ()
             message(FATAL_ERROR "Unknown build type: ${CMAKE_BUILD_TYPE}")
         endif ()
     endif ()
+
+    # Options definition
     option(BUILD_TESTS "If enabled, the SDK will include tests in the build" OFF)
 
+    message(STATUS "Setting toolchain and dependencies")
+    if (BUILD_TESTS)
+        message(STATUS "Building tests")
```

Comment:
unrelated: isn't [BUILD_TESTING](https://cmake.org/cmake/help/latest/command/enable_testing.html) the official cmake option?

Reply:
Good catch, updating.